schema.sql

```diff
@@ -0,0 +1,179 @@
+DROP DATABASE IF EXISTS lairdnd;
+CREATE DATABASE lairdnd;
+
+USE lairdnd;
+
+/* -------------------------------------------------------*/
+/*                      Create tables                     */
+/* -------------------------------------------------------*/
+
+CREATE TABLE photos (
+	id INT AUTO_INCREMENT PRIMARY KEY,
+	offerings_id INT,
+	url VARCHAR(255),
```

Comment:
This is a test comment.

Reply:
Hey Angela, I disagree as this is only a test. Will work on the rest of the script later. Please approvel

Reply:
Ok, looks good!